.github/workflows/tag-for-release.yml

```diff
@@ -1,20 +1,44 @@
 on:
   push:
-    branches:
-      - main
-
-permissions:
-  contents: write
-  pull-requests: write
-
-name: release-please
+    branches: [ main ]
 
 jobs:
-  release-please:
+  release-package:
     runs-on: ubuntu-latest
+    permissions: 
+      contents: write
+      pull-requests: write
+    outputs:
+      release-created: ${{ steps.release.outputs.release_created }}
+      upload-tag-name: ${{ steps.release.outputs.tag_name }}
```

Comment:
These outputs are only needed if we had a separate job that needed to reference them. Based on how you have it now, we could just get rid of them.

Alternatively, we could make the goreleaser stuff a separate job that only runs if a release is created. That might actually be a cleaner implementation because we wouldn't have to call `if: ${{ steps.release.outputs.releases_created == 'true' }}` in every step.